src/libmongoc/src/mongoc/mongoc-counters-private.h

```diff
@@ -101,6 +101,16 @@ _mongoc_sched_getcpu (void)
 }
 #elif defined(MONGOC_HAVE_SCHED_GETCPU)
 #define _mongoc_sched_getcpu sched_getcpu
+#elif defined(__APPLE__) && defined(__aarch64__)
```

Comment:
Is it possible that `tipdrro_el0` is not available on less privileged modes of execution?

If so, I suggest wrapping this in a conditional define, similar to `MONGOC_ENABLE_RDTSCP`.

Reply:
Thankfully it is accessible in non privileged mode (e.g. some memory allocators, etc use it).